src/Action/Admin/DownloadCreditMemoAction.php

```diff
@@ -20,22 +20,21 @@
 
 final class DownloadCreditMemoAction
 {
-    private CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator;
-
-    private CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder;
-
     public function __construct(
-        CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator,
-        CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder
+        private CreditMemoPdfFileGeneratorInterface $creditMemoPdfFileGenerator,
+        private CreditMemoFileResponseBuilderInterface $creditMemoFileResponseBuilder,
+        private ?string $hasEnabledPdfFileGenerator = null
```

Comment:
shouldn't be boolean? In the configuration, it is configured as a boolean.
why do we need to make it nullable? If I see correctly you've configured the default value as a true